lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -292,7 +289,7 @@ func (c *Client) startRustRdp(ctx context.Context) error {
 	}
 
 	// TODO(isaiah): better error handling here
```

Comment:
Do we need to keep the TODO or is this better?

Reply:
I think it's better now, I removed the TODO